kube/kubetypes/metrics.go

```diff
@@ -11,6 +11,7 @@ const (
 	AppIngressResource = "k8s-operator-ingress-resource"
 	AppEgressProxy     = "k8s-operator-egress-proxy"
 	AppConnector       = "k8s-operator-connector-resource"
+	AppProxyGroup      = "k8s-operator-proxy-group"
```

Comment:
This will be slightly different from other app types where the metric can be used to count how many kube resources of this type exists, but that's fine.

Reply:
Should we differentiate between egress and ingress here as well? Seems like it will be helpful once we ship HA ingress

Reply:
I don't feel strongly either way. The `MetricProxyGroupCount` that we added is not egress specific, so then we need to change that one too to be consistent.

Reply:
Changed both the app type and the metric to be egress specific.